app/models/media_resource.rb

```diff
@@ -101,4 +101,29 @@ def replace?(res)
   def update_resource(res)
     # NOTE: media_resources have no user settable fields
   end
+
+  def retryable?
+    if status_started? || status_created? || status_processing?
+      (Time.now - updated_at) > 30
+    else
+      false
+    end
+  end
+
+  def retry!
+    status_retrying!
+    copy_media(true)
+  end
+
+  def _retry=(_val)
```

Comment:
What does the underscore before the method name here do?

Reply:
I was just following a convention from rails forms/formbuilder, where you use `_destroy` to magically mark a record for destruction.  But here, I'm magically marking this file's processing to get retried.